src/Forms/GridField/GridFieldSortableHeader.php

```diff
@@ -147,7 +155,7 @@ public function getHTMLFragments($gridField)
                     if ($tmpItem instanceof SS_List) {
                         // It's impossible to sort on a HasManyList/ManyManyList
                         break;
-                    } elseif ($tmpItem && method_exists($tmpItem, 'hasMethod') && $tmpItem->hasMethod($methodName)) {
+                    } elseif ($tmpItem && ClassInfo::hasMethod($tmpItem, $methodName)) {
```

Comment:
Technically we only support ViewableData which will always have the `hasMethod` method.... but this is just way more straight forward than the existing check without introducing potential regressions in case someone was passing some non-ViewableData through here.